admin/services/access-arrangements.service.js

```diff
@@ -10,4 +16,65 @@ accessArrangementsService.getAccessArrangements = async () => {
   return accessArrangementsDataService.sqlFindAccessArrangements()
 }
 
-module.exports = accessArrangementsService
+/**
+ * Process access arrangements for single pupil
+ * @param {Object} requestData
+ * @param {Number} dfeNumber
+ * @param {Number} userId
+ * @returns {Object}
+ */
+accessArrangementsService.submit = async (requestData, dfeNumber, userId) => {
+  const validationError = accessArrangementsValidator.validate(requestData)
+  if (validationError.hasError()) {
+    throw validationError
+  }
+  const { pupilUrlSlug, accessArrangements: accessArrangementsCodes, questionReaderReason: questionReaderReasonCode } = requestData
+  let pupilAccessArrangements = R.clone(requestData)
```

Comment:
It's surprising to see `let` being used here when care is being taken to clone the input data.  Could this be const?

Reply:
@jon-shipley this pattern is now updated